### PR TITLE
Delete unused files

### PR DESCRIPTION
**Pull request by davidkokkelink**
This PR deletes a bunch of unused PHP, twig and JavaScript files.

### CODE REVIEW

File: controller/plugin-templates.php [hunk 1]
```diff
@@ -1,24 +1,5 @@
 <?php
```
Comment (gka) [inline controller/plugin-templates.php:1]: did you check that no single plugin is using these?

File: www/static/js/dw/chart/publish.js [hunk 2]
```diff
@@ -1,112 +0,0 @@
...
-            var w = $('#embed-width').val(), h = $('#embed-height').val(),
-                embedCode = $('#iframe-code').html();
-            embedCode = embedCode.replace(/width="\d+"/, 'width="' + w + '"');
-            embedCode = embedCode.replace(/height="\d+"/, 'height="' + h + '"');
-            $('#iframe-code').html(embedCode);
-
-            resizeIFrame();
-        });
-
-        $('.chart-actions .action-duplicate a').click(triggerDuplicate);
```
Comment (gka) [inline www/static/js/dw/chart/publish.js:20]: if you delete this, the chart duplication UI will stop working

File: controller/plugin-templates.php (hunk 1 above)
Comment (davidkokkelink) [reply controller/plugin-templates.php:1]: yes

File: www/static/js/dw/chart/publish.js (hunk 2 above)
Comment (davidkokkelink) [reply www/static/js/dw/chart/publish.js:20]: thanks, fixed

**Comment by davidkokkelink** [comment]
Thank you for the review, I have fixed the chart duplication bug. I did code searches for all the functionality that I removed, and tested locally to ensure that nothing breaks.

I know this is an annoying PR to review, but I think it's important that we do it:
- duplicate and dead code is confusing for new developers
- having effectively unmaintained code be executed server-side on our production systems is a security risk
- reducing the scope of this repository brings us closer to our goal of getting rid of PHP entirely

I have deployed this branch on staging for further testing.

**Comment by davidkokkelink** [comment]
This was running on staging for close to two weeks without problems now. Planning to merge this tomorrow unless any issues are raised.

**Comment by davidkokkelink** [comment]
and off to production it went

**Pull request merged**